tests/Api/Serializer/ComplianceTest.php

```diff
@@ -22,14 +22,39 @@ class ComplianceTest extends TestCase
 {
     use UsesServiceTrait;
 
+    public const TEST_CASES_DIR = __DIR__ . '/../test_cases/protocols/input/';
+
+    private static array $excludedCases = [
+        'RestJsonNullAndEmptyHeaders' => true,
```

Comment:
This and `NullAndEmptyHeaders` are blocked by Guzzle.  See https://github.com/guzzle/psr7/pull/625